e2e/client/playwright/page-object-models/authoring.ts

```diff
@@ -25,11 +25,19 @@ export class Authoring {
     field(field: string): Locator {
         return this.page.locator(s('authoring', field)).getByRole('textbox');
     }
+
+    async waitingForToastMsg(type: string, text: string): Promise<void> {
+        const selector = `notification--${type}=${text}`;
```

Comment:
Naming isn't the best here, since that string is the argument to `s` function, not actual CSS selector. It would be better to store the result of `s` - then you'd have a standard CSS selector - and would avoid calling `s` multiple times.